### PR TITLE
Minify Module Image Not Showing Fix

### LINKED ISSUE #1420: Minify-Image module not working
### Please describe the problem (or idea)
Minify Image module is not showing the output image.
> What happened just before the problem occurred? Or what problem could this idea solve?
bug

> What did you expect to see that you didn't?
I didnt saw any output image.

### Please show us where to look

https://beta.sequencer.publiclab.org

### What's your PublicLab.org username?

> This can help us diagnose the issue:

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by root00198**
Fixes #1420  (<=== Replace `0000` with the Issue Number)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/is-reviewers` for help, in a comment below
* [ ] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

### CODE REVIEW

**Comment by root00198** [comment]
I just realized that the module is not working correctly....

File: src/modules/MinifyImage/Module.js [hunk 1]
```diff
@@ -76,7 +78,7 @@ module.exports = function MinifyImage(options, UI) {
         });
         var destPath = __dirname + '/results/test.' + input.format;
         var data = base64Img.base64Sync(destPath);
-        output(data, input.format);
+        output(destPath, data, input.format);
```
Comment (HarshKhandeparkar) [inline src/modules/MinifyImage/Module.js]: Why do we do this in the browser? Why is it even required? Can't we just output the image? Was it used for debugging and was left out?

Comment (root00198) [reply src/modules/MinifyImage/Module.js]: This is used for command line interface.

Comment (HarshKhandeparkar) [reply src/modules/MinifyImage/Module.js]: But that part is handled by the sequencer core library code anyway.

**Pull request closed without merging**